Lib/fractions.py

```diff
@@ -770,7 +771,7 @@ def _add(a, b):
             return Fraction._from_coprime_ints(t, s * db)
         return Fraction._from_coprime_ints(t // g2, s * (db // g2))
 
-    __add__, __radd__ = _operator_fallbacks(_add, operator.add)
+    __add__, __radd__ = _operator_fallbacks(_add, operator.add, True)
```

Comment:
Not sure if this is a big deal, but this value fits most cases, isn't?  Maybe this should be a default?

Reply:
Indeed. This will help us keep the diff minimal.